docs/migration-guide/migration-guide-master-page.md

```diff
@@ -0,0 +1,48 @@
+5. [Verify the integrity of your project](#5-verify-the-integrity-of-your-project)
+6. [Customise your project folder structure](#6-customise-your-project-folder-structure)
+
+**Note:** Migrating a SpatialOS project from CPL to FPL is an all or nothing process. A project must either fully follow the CPL format or the FPL format and cannot have partially migrated components.
+
+**Tip:** We recommend that you back up a copy of your project before you start migrating so that you can recover information from files that you delete as part of the migration process if needed.
+
+## 1. Migrate your workers
+1. Migrate the build process of your workers by following the steps in [Building a worker executable](../build-process/worker-build-process.md).
+2. Migrate your workers’ worker configuration files by following the steps in [Converting the worker configuration file](configs-conversion-guide.md#converting-the-worker-configuration-file).
+3. Delete all previous CPL format configuration files related to the configuration and build process of your workers. Those configuration files include:
+    - Your CPL format [worker configuration files](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-configuration)
+    - Your CPL format [worker build configuration files](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-build)
+    - [spatialos_worker_packages.json files](https://docs.improbable.io/reference/latest/shared/reference/file-formats/spatial-worker-packages)
+
+## 2. Migrate your launch configuration files
+1. Migrate your launch configuration files by following the steps in [Converting the launch configuration file](configs-conversion-guide.md#converting-the-launch-configuration-file).
+2. Delete your previous CPL format launch configuration file.
+
+## 3. Migrate your project definition file
+1. Migrate your project definition file by following the steps in [Converting the project definition file](configs-conversion-guide.md#converting-the-project-definition-file).
+2. Delete your previous CPL format project definition file.
+
+## 4. Generate a schema descriptor
+Generate a schema descriptor by following the steps in [Building a schema descriptor](../build-process/schema-descriptor-build-process.md).
+
+## 5. Verify the integrity of your project
+Your project is now in a state where it complies with the FPL and can be run.
+- Run `spatial alpha local launch` and verify that your deployment behaves how it did pre-migration.
+- Run `spatial alpha cloud upload -a <some-assembly-name>` and verify that this uploads your project assembly correctly to `https://console.improbable.io/projects/<your-project-name>/assemblies`. Your project assemblies should contain your schema descriptor and a zip file for each worker and each supported platform in your project. Each worker zip file should contain the content of the `localBuildDirectory` folder of your worker configurations.
```

Comment:
```suggestion
- Run `spatial alpha cloud upload -a <some-assembly-name>` and verify that this uploads your project assembly correctly to `https://console.improbable.io/projects/<your-project-name>/assemblies`. Your project assemblies should contain your schema descriptor and a zip file for each worker and each supported platform in your project. Each worker zip file should contain the content from the `localBuildDirectory` folder of the worker configuration file.
```
Tried to make this clearer, but please check that my updates are correct.

Reply:
Done.